packages/atlas/src/components/FiltersBar/FiltersBar.stories.tsx

```diff
@@ -21,16 +23,24 @@ import { useFiltersBar } from './useFiltersBar'
 export default {
   title: 'other/FiltersBar',
   component: FiltersBar,
+  args: {
+    activeFilters: ['date', 'other', 'categories', 'length', 'nftStatus', 'language'],
+  },
   decorators: [
-    (Story) => (
-      <OverlayManagerProvider>
-        <Story />
-      </OverlayManagerProvider>
-    ),
+    (Story) => {
+      const apolloClient = createApolloClient()
+      return (
+        <ApolloProvider client={apolloClient}>
+          <OverlayManagerProvider>
+            <Story />
+          </OverlayManagerProvider>
+        </ApolloProvider>
+      )
+    },
   ],
 } as Meta
 
-const RegularTemplate: Story<FiltersBarProps> = () => {
+const RegularTemplate: Story<FiltersBarProps> = (args) => {
```

Comment:
Could you clean this story up? Currently it has unnecessary stuff there other than just the filters bar and also it seems the filters are unusable. When I click on specific filters I don't get any popup